cmd/promtool/main.go

```diff
@@ -715,9 +715,22 @@ func CheckRules(ls lintConfig, files ...string) int {
 }
 
 func checkRules(filename string, lintSettings lintConfig) (int, []error) {
-	fmt.Println("Checking", filename)
+	var rgs *rulefmt.RuleGroups
+	var errs []error
+	if filename == "-" || filename == "" {
+		var buf bytes.Buffer
+		tee := io.TeeReader(os.Stdin, &buf)
```

Comment:
I think this was copied from `CheckMetrics`, this shouldn't need TeeReader, just `io.ReadAll` is enough.